src/notes_handlers.py

```diff
@@ -49,6 +53,10 @@ def all_notes(notes:NoteBook) -> str:
     if not notes.data:
         return INFO + " You do not have any notes saved"
     res=""
-    for note in notes.values():
-        res += str(note)+"\n"+ "-"*30 + "\n"
-    return res.strip()
+    for title, note in notes.items():
+        res += "---------------------------------------------------------------------------------------------\n"
```

Comment:
and if we are here anyway, would be better to change this to "-"*80 (or whatever number)

Reply:
I wrote that i had surprised coz I could see your review online in my VSC. It's was for the first time:)

Reply:
Thank you. Done.